mmv1/third_party/terraform/services/storage/resource_storage_bucket_object.go

```diff
@@ -91,6 +91,7 @@ func ResourceStorageBucketObject() *schema.Resource {
 				ForceNew:     true,
 				ExactlyOneOf: []string{"source"},
 				Sensitive:    true,
+				Computed:     true,
```

Comment:
I'm not familiar with the internal of the provider implementation so correct me I'm wrong. :bow:

This patch seems to update the `google_storage_bucket_object` **resource**, not the `google_storage_bucket_object_content` **data source**.
The `content` of the `google_storage_bucket_object` **resource** is supposed to be filled by users, as noted in the attribute description: "Data as string to be uploaded. Must be defined if source is not.".

I think we should update `data_source_storage_bucket_object_content.go` to fix https://github.com/hashicorp/terraform-provider-google/issues/15717.

Reply:
From the [code](https://github.com/hashicorp/terraform-provider-google/blob/c4b0f12b4ce7a09292fb3bb9e71e9713655f7129/google/services/storage/data_source_storage_bucket_object_content.go#L19), you may see they share the same schema

Reply:
The `content` attribute of the data source is marked as non-computed immediately after the line you referenced, as I noted in the original issue.

https://github.com/hashicorp/terraform-provider-google/blob/c4b0f12b4ce7a09292fb3bb9e71e9713655f7129/google/services/storage/data_source_storage_bucket_object_content.go#L23